project/Common.scala

```diff
@@ -47,13 +53,22 @@ object Common extends AutoPlugin {
         }
       }
     },
-    scalaVersion := "2.12.17",
-    crossScalaVersions := Seq(
-      "2.11.12",
-      scalaVersion.value,
-      "2.13.10",
-      "3.2.2"
-    ),
+    // Play 3 not supported in 2.11/2.12
+    scalaVersion := "2.13.11",
```

Comment:
Rather keep 2.12 as lower default, and condition for other case

Reply:
I have reverted the change, it was not possible to use the play version because of cyclic dependencies. Since the scala version is hardcoded in the build and the testing process i though i could just keep it as it was.
I have upgraded the version of scala 2.13.10 to 2.13.11 and scala 3.2.2 to 3.3.3 to align test and build.